src/CarbonAware.WebApi/test/integrationTests/IntegrationTestingBase.cs

```diff
@@ -18,7 +18,8 @@ namespace CarbonAware.WebApi.IntegrationTests;
 public abstract class IntegrationTestingBase
```

Comment:
In VS Code this file has some missing references (Program) and will not compile.